### ISSUE TITLE
Search on field that contains array items

### ISSUE THREAD

**Issue by primarius27** [react-bootstrap-table/react-bootstrap-table2#642]
When you try to search on column that contains array items it's not working properly. Is there any way to fix this issue? 
Example: array = [ 'dog', 'cat', 'parrot'] 
search field value = 'dog parrot' 
Result: row is hidden which should not be the case.

Thanks.

**Comment by primarius27** [comment]
Also to add, is it possible to search on multiple columns in same time?

**Comment by AllenFang** [comment]
yeah... currently we just support the text search. for the array search, I will consider to implement, but not now. Thanks
